test/test_sqlalchemy_bigquery.py

```diff
@@ -199,9 +199,7 @@ def test_engine_with_dataset(engine_using_test_dataset):
     rows = table_one_row.select().execute().fetchall()
     assert list(rows[0]) == ONE_ROW_CONTENTS_EXPANDED
 
-    # Table name shouldn't include dataset
-    with pytest.raises(Exception):
-        table_one_row = Table('test_pybigquery.sample_one_row', MetaData(bind=engine_using_test_dataset), autoload=True)
+    table_one_row = Table('test_pybigquery.sample_one_row', MetaData(bind=engine_using_test_dataset), autoload=True)
```

Comment:
probably better to check that the expected results is returned

Reply:
Could we add an assertion on some expected property of the Table? In this example, we want to make sure that the dataset ID associated with the table == "test_pybigquery" and != whatever the default dataset ID is, right?